r/R/dataset-write.R

```diff
@@ -56,11 +56,12 @@
 #' @export
 write_dataset <- function(dataset,
                           path,
-                          format = dataset$format,
+                          format = c("parquet", "feather", "arrow", "ipc"),
```

Comment:
Need to update the docs to reflect the new behavior. You may (also) want to borrow the language used to describe `format` from `dataset-format.R`.

Reply:
how about this?
![Uploading image.png…]()

Reply:
xref to FileFormat is good (if it works, does it need the `()`? FileFormat is not callable) though I think it's important to (still) note that only parquet and feather are supported.

Reply:
fixed! the link it works without ()
![image](https://user-images.githubusercontent.com/10091065/109722648-bc044880-7b8b-11eb-9078-2deff6ce1450.png)